### DISCUSSION TITLE
smal changes

### DISCUSSION THREAD

**Discussion by forenade** [kimai/kimai#3084] [Questions & Answers]
Kimai works great for me but i wont to make same small changes. 

First i need to add a new activity that make my people able to add what they drive in Km (miles), so they can report the km to work as a activity.

Then i wonder if there is posible to make a new tab on the left side  with the dashbord and other that that show a .pdf or. .doc file that i can edit and show info to my employees.  With pic and text.

Is these possible ?

Thanks for a great software your are doing a great work..

**Comment by kevinpapst** [comment]
Reporting expenses (eg. driven kilometers) should done with the Expense plugin: 
https://www.kimai.org/store/expenses-bundle.html

> Then i wonder if there is posible to make a new tab on the left side with the dashbord and other that that show a .pdf or. .doc file that i can edit and show info to my employees. With pic and text.

I don't understand this one.

**Reply by forenade** [reply]
Ok great will look in to that. 

I mean if i can add a tab on the left side where i can add a news letter to my guys that i can change every month?

**Reply by kevinpapst** [reply]
Interesting question, that nobody ask for until now. But yes, possible.
I am working on a new plugin for custom content (css, javascript broadcast alert). this can be used to add a custom page as well. 
The plugin will be released with the next major version 1.17. I try to remember to post an update here once it is ready.

**Reply by forenade** [reply]
Great that sounds like it could solv my problem. 👌

will that make a tab on the leftside that i can redirect to custom page ?

**Reply by kevinpapst** [reply]
No, it creates a menu entry (one the left side) which links to a page rendering markdown content.
See screenshots: https://www.kimai.org/store/custom-content-bundle.html
You can test it in the "Plugins demo".
